h/static/scripts/group-forms/components/ModerationStatusSelect.tsx

```diff
@@ -0,0 +1,60 @@
+import type { IconComponent } from '@hypothesis/frontend-shared';
+import { FilterIcon } from '@hypothesis/frontend-shared';
+import { CautionIcon } from '@hypothesis/frontend-shared';
+import { RestrictedIcon } from '@hypothesis/frontend-shared';
+import { CheckAllIcon } from '@hypothesis/frontend-shared';
+import { DottedCircleIcon, Select } from '@hypothesis/frontend-shared';
+
+export type ModerationStatus = 'pending' | 'approved' | 'denied' | 'spam';
+
+export type ModerationStatusSelectProps = {
+  selected?: ModerationStatus;
+  onChange: (status?: ModerationStatus) => void;
+};
+
+type Option = {
+  icon: IconComponent;
+  name: string;
```

Comment:
Perhaps `label` rather than `name` to make it clear this is a text label in the UI rather than an identifier.